radicle/src/identity/doc.rs

```diff
@@ -174,6 +178,24 @@ impl Doc<Verified> {
         false
     }
 
+    pub fn rescind(&mut self, key: &crypto::PublicKey) -> Result<Option<Did>, DocError> {
+        let delegate = Did::from(key);
+        let (matches, delegates) = self.delegates.iter().partition(|d| **d == delegate);
+        match NonEmpty::from_vec(delegates) {
+            Some(delegates) => {
+                self.delegates = delegates;
+                if self.threshold > self.delegates.len() {
+                    return Err(DocError::Threshold(
+                        self.threshold,
+                        "the thresholds exceeds the new amount of delegates after removal",
```

Comment:
```suggestion
                        "the thresholds exceeds the new delegate count after removal",
```
"amount" made it sound like we're measuring something :)